### PR TITLE
add RangeStream test for values exceeding chunk target

### PR DESCRIPTION
**Pull request by Jefftree**
Could be an edge case where we have 4MB values or something. Ensure that RangeStream still progresses despite each chunk exceeding the preconfigured 1.5MB chunk limit.

### CODE REVIEW

**Review by serathius — APPROVED** [review]

File: tests/integration/v3_grpc_test.go [hunk 1]
```diff
@@ -1845,6 +1829,66 @@ func TestV3RangeStreamWriteBetweenChunks(t *testing.T) {
...
+		_, err := kvc.Put(t.Context(), &pb.PutRequest{
+			Key:   []byte(fmt.Sprintf("big-%02d", i)),
+			Value: value,
+		})
+		require.NoError(t, err)
+	}
+
+	m := clus.Members[0]
+	m.Stop(t)
+	m.MaxRequestBytes = embed.DefaultMaxRequestBytes
```
Comment (fuweid) [inline tests/integration/v3_grpc_test.go:1862]: maybe we should make sure that DefaultMaxRequestBytes is small than 4 MiB at the beginning.

Comment (Jefftree) [reply tests/integration/v3_grpc_test.go:1862]: https://github.com/etcd-io/etcd/blob/main/server/embed/config.go#L67. Do you mean explicitly asserting it?

Comment (Jefftree) [reply tests/integration/v3_grpc_test.go:1862]: Added `require.Less(t, int(m.MaxRequestBytes), valueSize, "valueSize must exceed MaxRequestBytes to exercise large-value path")`

**Review by fuweid — APPROVED** [review]
